src/docs/user-guide/running-jobs.md

```diff
@@ -675,7 +675,7 @@ restarts for 5 minutes, and so on, until it's properly `scancel`led.
 [url_bash]: https://www.gnu.org/software/bash/manual/bash.html
 [url_scancel]: https://slurm.schedmd.com/scancel.html
 
-[url_ruse]:     https://github.com/JanneM/Ruse
+[url_ruse]:     https://github.com/JanneM/
```

Comment:
Not sure about this one? The project URL seemed correct?

Reply:
ya I added an extra space by mistake while comtrol-F'in :)